test/linux/slaveinfo/slaveinfo.h

```diff
@@ -0,0 +1,12 @@
+/** \file
+ * \brief Declaration of slaveinfo for SOEM.
```

Comment:
Could you motivate why this is being extracted? Do you intend to use it outside of SOEM? Currently this file is not referenced anywhere in the tree as far as I can tell. It should probably at least be included by slaveinfo.c

Reply:
I was running some tests with RTEMS. Possibly I should clean up those and push them as well as a separate RTEMS tests.